code/HumanGEMInstaller.m

```diff
@@ -0,0 +1,44 @@
+classdef HumanGEMInstaller
+% HumanGEMInstaller
+%   Support for installing and uninstalling
+%   Run HumanGEMInstaller.install to install (will set up the path in MATLAB)
+%   Run HumanGEMInstaller.uninstall to clear the path from MATLAB
+%   To install, you first need to cd to the HumanGEM root.
+
+    methods (Static)
+        function install
+            sourceDir = fileparts(which(mfilename));
+            paths = HumanGEMInstaller.GetFilteredSubPaths(sourceDir);
+            addpath(paths);
+            savepath;
+        end
+        function uninstall
+            sourceDir = fileparts(which(mfilename));
+            paths = HumanGEMInstaller.GetFilteredSubPaths(sourceDir);
+            rmpath(paths);
+            savepath;
+        end
+        function path = getHumanGEMMainPath()
+			path = fileparts(which(mfilename));
+			path = strrep(path, '\', '/'); %get rid of backslashes in Windows
+			if ~endsWith(path, '/')
+				path = strcat(path,'/');
+			end
+		end
+
+        function newPaths = GetFilteredSubPaths(path_)
+            pathSep = pathsep();
+			%Check that there are no separators in the path - that will cause 
+            %problems since the separator is used to separate paths in a string
+			if contains(path_, pathSep)
+				error('The path in which Human-GEM resides may not contain path separator chars such as semicolon for this installation to work!');
+			end
+            subpaths = arrayfun(@(subf) sprintf('%s', genpath([path_ filesep subf{:}])), HumanGEMInstaller.SUBFOLDERS, 'UniformOutput', false);
+            newPaths = [strjoin(subpaths, '') pathSep path_];
```

Comment:
The installer file was moved and documented in [2e3f01c](https://github.com/SysBioChalmers/Human-GEM/pull/423/commits/2e3f01cf6ac345a0909f53c674fd80beea9b3764). The reason to go ahead was that we need to be mindful that users might not use Matlab, or care about the `code` in this repository at all, so a clean top-level folder is more appealing to them. Now, the previous fix is not longer required and could be reverted:
```suggestion
            newPaths = strjoin(subpaths, '');
```